source/develop/release-management/features/cloud/cloud-init-integration.html.md

```diff
@@ -60,9 +62,9 @@ TODO: still pending is whether and/or how to communicate to oVirt engine that cl
 There is a long list of configuration options supported by cloud-init. The ones we are currently targeting for inclusion in oVirt include:
 
 *   set root password
-*   set timezone\*
+*   set timezone
```

Comment:
It's actually 'timezone' - see https://github.com/number5/cloud-init/blob/master/doc/examples/cloud-config.txt#L382